app/views/posts/index.html.erb

```diff
@@ -16,7 +16,7 @@
             </div>
         </div>
     <% end %>
-    <div class="d-flex flex-column p-3 mb-5 bg-body rounded">
+    <div class="align-self-start d-flex flex-column p-3 mb-5 bg-body rounded">
         <% post.most_recent_comments.each do |comment|%>
             <span> <%= comment.author.name %>: <%= comment.text%> </span>
         <% end %>

```

Comment:
- [ ] It looks like the design wasn't resolved as per request from the previous reviewer. Since the parent element has display flex, you can do `margin-right: auto` on the span element and the issue will be fixed.

- [ ] [OPTIONAL] The Microverse logo should be removed or replaced with your own if you have any.

Reply:
Done